deploy/README.md

```diff
@@ -44,4 +44,14 @@ To see all resources installed by the helm chart
 
 ```sh
 kubectl get elastic -l "app.kubernetes.io/instance"=es-kb-quickstart -n elastic-stack
-```
\ No newline at end of file
+```
+
+## Licensing
+
```

Comment:
That's fine but I think I should have mentioned the base license. Something along these lines:

```
All ECK Helm Charts are licensed under Elastic License 2.0 like the operator,
but require different subscription levels.

The ECK Operator Helm Chart can be used with a Basic license for free,
while the ECK Stack and Resources Helm Charts require an Enterprise license.
```